### ISSUE TITLE
new data link - MNDOT

### ISSUE THREAD

**Issue by mel1877** [Sai83/accessibilityscore#4]
http://mn.gov/mmb/mn-dashboard/mobility/public-transportation-use/index.jsp.

I'm not sure if this is useable data.  It is a historical look of transportation usage across the state.  Although it does look like additional data is getting posted on occasion.  Something to keep an eye on.

**Comment by diavy** [comment]
Yes this is very good data source.. I also checked the Google challenge, it looks very exciting but according to the instruction:only non-profit organizations or charities are eligible to submit an idea. Not sure if we’re eligible for that….

I think now we should focus on the city challenge and we have only one month to work. Did you guys have chance to check the proposed demo yet? I attach it here. Let’t try to reach an aggrement about the demo and have a group discuss soon (including Ryan, the guy who is very supportive of our joining the challenge). So we will be clear about whether our demo will meet the requirement of the challenge, and what other data we may need or request. 

Let’s try our best to meet (online) early next week if possible, since both Frank and Ryan will possibly leave later next week. Let me know what you guys think about it..
